build/BundledRuntimes.props

```diff
@@ -20,7 +20,8 @@
          NOTE: There isn't a 'linux-x64' version of the installers or the additional shared framweork. -->
     <SharedFrameworkRid>$(CoreSetupRid)</SharedFrameworkRid>
     <SharedFrameworkRid Condition=" '$(UsePortableLinuxSharedFramework)' == 'true' ">linux-x64</SharedFrameworkRid>
-    <CombinedFrameworkHostCompressedFileName>dotnet-runtime-$(SharedFrameworkVersion)-$(SharedFrameworkRid)$(ArchiveExtension)</CombinedFrameworkHostCompressedFileName>
+    <CombinedFrameworkHostCompressedFilePrefix>dotnet-runtime</CombinedFrameworkHostCompressedFilePrefix>
+    <CombinedFrameworkHostCompressedFileName>$(CombinedFrameworkHostCompressedFilePrefix)-$(SharedFrameworkVersion)-$(SharedFrameworkRid)$(ArchiveExtension)</CombinedFrameworkHostCompressedFileName>
```

Comment:
Thank you, this is an mistake. At one point I thought dotnet runtime rpm package does not have version on the end, I was wrong in the end.